### PR TITLE
fix(suite): edit settings layout

### PR DESCRIPTION
**Pull request by dimasivan**
Design: [Zeplin](https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/screen/5faaa48ab6e7968540370da5)

- Edit settings layout max-width to 768px
- Change settings button type to terciary on 'Learn more' buttons
- Add settings description max-width 500px

### CODE REVIEW

File: packages/suite/src/components/suite/Settings/components/TextColumn.tsx [hunk 1]
```diff
@@ -37,9 +45,14 @@ const TextColumn = ({ title, description, learnMore }: TextColumnProps) => (
         {title && <Title>{title}</Title>}
         {description && <Description>{description}</Description>}
         {learnMore && (
-            <ExternalLink href={learnMore} size="tiny">
-                <Translation id="TR_LEARN_MORE" />
-            </ExternalLink>
+            <LearnMoreLink
```
Comment (szymonlesisz) [inline packages/suite/src/components/suite/Settings/components/TextColumn.tsx]: do not use `styled.a`

related to https://github.com/trezor/trezor-suite/issues/3633

Comment (dimasivan) [reply packages/suite/src/components/suite/Settings/components/TextColumn.tsx]: I replaced it with TrezorLink component
Seems like #3633 is already in progress?

**Review by goodhoko — APPROVED** [review]
lgtm

File: packages/suite/src/components/suite/Settings/components/TextColumn.tsx (hunk 1 above)
Comment (szymonlesisz) [reply packages/suite/src/components/suite/Settings/components/TextColumn.tsx]: i dont think so, i didnt have a time to answer on that :)

**Review by keraf — APPROVED** [review]
Looks good! Please squash your last commit :slightly_smiling_face:

File: packages/suite/src/components/settings/SettingsLayout/index.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ const Wrapper = styled.div`
     display: flex;
     flex: 1;
     flex-direction: column;
+    max-width: 768px;
```
Comment (tomexx) [inline packages/suite/src/components/settings/SettingsLayout/index.tsx]: let's take this from global variables

```
import { variables } from '@trezor/components';
```

then use `variables.SCREEN_SIZE.MD`

File: packages/suite/src/components/settings/SettingsMenu/index.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ const SettingsMenu = () => {
 
     return (
         <AppNavigationPanel
-            maxWidth="default"
+            maxWidth="768px"
```
Comment (tomexx) [inline packages/suite/src/components/settings/SettingsMenu/index.tsx]: also here

**Review by tomexx — CHANGES_REQUESTED** [review]
pls check comments

File: packages/suite/src/components/settings/SettingsLayout/index.tsx (hunk 2 above)
Comment (dimasivan) [reply packages/suite/src/components/settings/SettingsLayout/index.tsx]: done, rebased

File: packages/suite/src/components/settings/SettingsMenu/index.tsx (hunk 3 above)
Comment (dimasivan) [reply packages/suite/src/components/settings/SettingsMenu/index.tsx]: done, rebased

**Pull request merged**